capstone/src/main/webapp/script.js

```diff
@@ -37,5 +39,36 @@ function getClubInfo() {
 
     document.getElementById('members').innerHTML = '# of Members: ' + clubInfo['members'].length;
     document.getElementById('website').innerHTML = 'Website: ' + clubInfo['website'];
-  });   
+  });
 }
+
+/** Accesses and displays club announcement data from servlet. */
+async function loadAnnouncements () {
+  const query = '/announcements';
+  const response = await fetch(query);
+  const json = await response.json();
+   const announcementsSection = document.getElementById('announcements-display');
+  announcementsSection.innerHTML = '<h1>Announcements</h1>';
+  announcementsSection.innerHTML += '<ul>';
+  for (var index in json) {
+      announcementsSection.innerHTML += '<li>'+json[index]+'</li>';
+  }
+  announcementsSection.innerHTML += '</ul>';
+}
+
+/** Displays club info tab, depending on which tab user selected. */
+function showTab(tabNum) {
+  if (tabNum == 1) {
```

Comment:
Is it possible to switch tabNum to tabName (with the name of the tab)? I think it'll make it more clear which tab we are opening, especially since there is no indication for the about-us tab. Up to you though, since it's a small change :)